charts/ratify/templates/verifier.yaml

```diff
@@ -13,19 +13,20 @@ spec:
   artifactTypes: application/vnd.cncf.notary.signature
   parameters:
     verificationCertStores:
-      certs:
```

Comment:
since it might be a breaking change during upgrading, we can leave this file unchanged, but only update the smaples and test files.

Reply:
updated